WeScan/ImageScannerController.swift

```diff
@@ -62,10 +73,30 @@ public final class ImageScannerController: UINavigationController {
         fatalError("init(coder:) has not been implemented")
     }
     
+    private func setupConstraints() {
+        let blackFlashViewConstraints = [
+            blackFlashView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            blackFlashView.topAnchor.constraint(equalTo: view.topAnchor),
+            view.trailingAnchor.constraint(equalTo: blackFlashView.trailingAnchor),
+            view.bottomAnchor.constraint(equalTo: blackFlashView.bottomAnchor)
+        ]
+        
+        NSLayoutConstraint.activate(blackFlashViewConstraints)
+    }
+    
     override public var supportedInterfaceOrientations: UIInterfaceOrientationMask {
         return .portrait
     }
     
+    internal func flashToBlack() {
+        view.bringSubview(toFront: blackFlashView)
+        blackFlashView.isHidden = false
+        let flashDuration = DispatchTime.now() + 0.05
+        DispatchQueue.main.asyncAfter(deadline: flashDuration) {
+            self.blackFlashView.isHidden = true
```

Comment:
Aren't you causing a retain cycle here by capturing self strongly?

Reply:
This was asked by Twan as well! 
I don't think so, since the closure is guaranteed to be called, and therefore release self.
Do you agree @renssies?

Reply:
Ok I just wanted to make sure :)